catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -74,32 +96,54 @@ public ContentDirectoryMonitor(CamelContext camelContext) {
      * also called whenever an existing route is updated.
      */
     public void init() {
-        LOGGER.trace("INSIDE: init()");
-
         if (routeCollection != null) {
             try {
                 // This stops the route before trying to remove it
                 LOGGER.debug("Removing {} routes", routeCollection.size());
                 camelContext.removeRouteDefinitions(routeCollection);
             } catch (Exception e) {
-                LOGGER.info("Unable to remove Camel routes from Content Directory Monitor", e);
+                LOGGER.warn("Unable to remove Camel routes from Content Directory Monitor", e);
             }
         } else {
             LOGGER.debug("No routes to remove before configuring a new route");
         }
 
-        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
-            configureCamelRoute();
+        runAsAdmin(() -> {
+            if (StringUtils.isEmpty(monitoredDirectory)) {
```

Comment:
Having a separate private method for this instead of inlining the code was nice. We should even be able to use a method reference instead: `runAsAdmin(this::configureCamelRoute)`.